lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1077,6 +1141,40 @@ def test_membership_staff_creator(self, user):
 
         self.assertEqual(team['membership'], [])
 
+    @ddt.unpack
+    @ddt.data(
+        ('student_enrolled', 400, 'You are already in a team in this course.'),
+        ('student_unenrolled', 403, None),
+        ('student_enrolled_not_on_team', 403, "You can't create a team in an instructor managed topic."),
+        ('student_masters', 400, 'You are already in a team in this course.'),
+        ('student_on_team_1_private_set_1', 400, 'You are already in a team in this course.'),
+        ('student_on_team_2_private_set_1', 400, 'You are already in a team in this course.'),
+        ('staff', 200, None)
+    )
+    def test_private_managed_access(self, user, expected_response, msg):
+        """
+        As different users, check if we can create a team in a private teamset.
+        Only staff should be able to crate teams in managed teamsets, but they're also
```

Comment:
nit - typo: create not crate.

Reply:
thank you, and please point out any other typos, my brain was complete slush by the time I was done typing all of these xD